CHANGELOG.md

```diff
@@ -21,12 +21,13 @@ The types of changes are:
 ### Added
 * [Admin UI] Update Subject Request status filter to be a multiselect dropdown [#513](https://github.com/ethyca/fidesops/pull/764)
 * Add support for multiple statuses to be selected for filtering subject requests [#660](https://github.com/ethyca/fidesops/pull/802)
-* Erasure support for Zendesk [#775](https://github.com/ethyca/fidesops/pull/775)
+* [Datastore Management] Include Icon with datastore connection (FE)[#671](https://github.com/ethyca/fidesops/pull/845)
```

Comment:
any particular reason for re-ordering these changelog items?

Reply:
Just sorted by Ticket #. No specific reason. Just remember Paul mentions he does it.